Getting-Started.md

```diff
@@ -709,4 +853,58 @@ Kibiter then automatically generates a query as shown below.
 
 > :warning: **If you change the structure or delete it** you won't see the menu.
 
-> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
\ No newline at end of file
+> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
+
+#### Dump a mapping/data from an index [&uarr;](#dump-data/mapping-)
+
+**elasticdump installation**
```

Comment:
Please add a short paragraph to let the user understand the overall process. For instance:

```
Indexes and mappings can be dumped (or uploaded) using [elasticdump](link to the doc). The commands to install elasticdump, dump an index and its index pattern to a file are shown below.  
```

Reply:
Okay..Thanks! Will add this.